internal/ui/src/components/id.tsx

```diff
@@ -0,0 +1,65 @@
+"use client";
+import { TaskChecked, TaskUnchecked } from "@unkey/icons";
+import * as React from "react";
+import { cn } from "../lib/utils";
+import { Tooltip, TooltipContent, TooltipTrigger } from "./tooltip";
+
+type IdProps = {
+  value: string;
```

Comment:
can you add docstrings to these?
that way your editor shows you inline documentation about each prop

```tsx
/**
* Explanation...
*/
truncate?: number
``